src/Stripe.net/Services/SubscriptionSchedules/SubscriptionSchedulePhaseOptions.cs

```diff
@@ -7,131 +7,56 @@ namespace Stripe
-        /// Either <c>charge_automatically</c>, or <c>send_invoice</c>. When
-        /// charging automatically, Stripe will attempt to pay the underlying
-        /// subscription at the end of each billing cycle using the default
-        /// source attached to the customer. When sending an invoice, Stripe
-        /// will email your customer an invoice with payment instructions.
-        /// Defaults to <c>charge_automatically</c> on creation.
-        /// </summary>
+        public SubscriptionSchedulePhaseBillingThresholdsOptions BillingThresholds { get; set; }
+
         [JsonProperty("collection_method")]
         public string CollectionMethod { get; set; }
 
-        /// <summary>
-        /// The code of the coupon to apply to this subscription. A coupon applied to a
-        /// subscription will only affect invoices created for that particular subscription.
-        /// </summary>
         [JsonProperty("coupon")]
         public string Coupon { get; set; }
 
-        /// <summary>
-        /// ID of the default payment method for the subscription schedule.
-        /// </summary>
         [JsonProperty("default_payment_method")]
         public string DefaultPaymentMethod { get; set; }
 
-        /// <summary>
-        /// Ids of the tax rates to apply to this phase on the subscription schedule.
-        /// </summary>
         [JsonProperty("default_tax_rates")]
         public List<string> DefaultTaxRates { get; set; }
 
-        /// <summary>
-        /// The date at which this phase of the subscription schedule ends. If set,
-        /// <c>iterations</c> must not be set.
-        /// </summary>
         [JsonProperty("end_date")]
         [JsonConverter(typeof(UnixDateTimeConverter))]
         public DateTime? EndDate { get; set; }
 
-        /// <summary>
-        /// Define the default settings applied to invoices created by this subscription schedule.
-        /// </summary>
         [JsonProperty("invoice_settings")]
-        public SubscriptionScheduleInvoiceSettingsOptions InvoiceSettings { get; set; }
-
-        /// <summary>
-        /// Integer representing the multiplier applied to the plan interval. For example,
-        /// <c>iterations=2</c> applied to a plan with <c>interval=month</c> and
-        /// <c>interval_count=3</c> results in a phase of duration 2 * 3 months = 6 months.
-        /// If set, <c>end_date</c> must not be set.
-        /// </summary>
+        public SubscriptionSchedulePhaseInvoiceSettingsOptions InvoiceSettings { get; set; }
+
         [JsonProperty("iterations")]
         public long? Iterations { get; set; }
 
-        /// <summary>
-        /// List of configuration items, each with an attached plan, to apply during this phase of
-        /// the subscription schedule.
-        /// </summary>
         [JsonProperty("plans")]
-        public List<SubscriptionSchedulePhaseItemOptions> Plans { get; set; }
+        public List<SubscriptionSchedulePhasePlanOptions> Plans { get; set; }
```

Comment:
Same as above, it's likely worth doing an override because we are renaming `Plans` to `Items` soon